entrypoint.sh

```diff
@@ -15,28 +15,38 @@ if [ ! -e $image_path ]; then
   fi
 fi
 
+echo "Rounding image size up to a multiple of 2G"
+image_size=`du -m $image_path | cut -f1`
```

Comment:
Are you sure this calculation is correct?

If I run:

```
docker run -it -v /tmp/dockerpi:/sdcard dockerpi
```

Then the image gets persisted at `/tmp/dockerpi/filesystem.img`.

However dockerpi attempts to resize it on every boot saying:

```
Rounding image size up to a multiple of 2G
from 2145M to 4G
```

This command seems to disagree with `ls`:

```
$ du -m /tmp/dockerpi/filesystem.img | cut -f1
2145

$ ls -lah /tmp/dockerpi/filesystem.img
-rw-r--r--@ 1 lukechilds  wheel   4.0G 16 May 15:45 /tmp/dockerpi/filesystem.img
```

If the image is already an appropriate size, we shouldn't attempt to resize it again.

Reply:
The resize is near instant, it's just a change to the filesize so the image become a sparse file. So it shouldn't even consume any more storage. But if you can think of a better way to do it.

Reply:
Sooo after a some investigating it looks like the original code was checking size on disk but what QEMU is checking is the virtual size:

```
$ qemu-img info /sdcard/filesystem.img
image: /sdcard/filesystem.img
file format: raw
virtual size: 4 GiB (4294967296 bytes)
disk size: 2.09 GiB
```

Resolved in https://github.com/lukechilds/dockerpi/pull/21/commits/5020c6394f49598b4fefa266b39ed34f79841504.